### ISSUE TITLE
Cell cycle info of each single cell

### ISSUE THREAD

**Issue by Avon0905** [SlavovLab/plexDIA#4]
Hi, cell ids in Proteins_SC.txt are different with those in Meta_SingleCell.tsv.  Can you help me with the cell cycle info of each single cell in Proteins_SC.txt file? How to match these cell id with Meta file's cell id?

**Comment by j-derks** [comment]
Hi! I just updated the the [single-cell directory](https://drive.google.com/drive/folders/1pUC2zgXKtKYn22mlor0lmUDK0frgwL_-) with a new meta data file that specifies an id for each cell "Meta_SingleCell_updated_1.tsv" and uploaded a new protein x single-cell matrix called "Proteins_SC_IDs.txt" that now has the cell-type header replaced with a unique id that can be used to trace it back to the meta data. Thanks for letting me know about this. 

As for the cell-cycle data, I have not done cell-cycle analysis on our single cells. The cell cycle analysis we performed in the plexDIA publication was on bulk U-937 samples.
